src/go/rpk/pkg/cli/registry/schema/schema.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package schema
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/sr"
+)
+
+func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "schema",
+		Args:  cobra.ExactArgs(0),
+		Short: "Manage your schema registry schemas",
+	}
+	cmd.AddCommand(
+		newCheckCompatibilityCommand(fs, p),
+		newCreateCommand(fs, p),
+		newDeleteCommand(fs, p),
+		newGetCommand(fs, p),
+		newListCommand(fs, p),
+	)
+	p.InstallFormatFlag(cmd)
+	return cmd
+}
+
+type subjectSchema struct {
+	Subject string `json:"subject" yaml:"subject"`
+	Version int    `json:"version" yaml:"version"`
+	ID      int    `json:"ID" yaml:"ID"`
```

Comment:
Is uppercase ID normal in json? I feel like it's normally lowercase -- I'd err on lowercase here.

Reply:
Second this, json is conventionally lowercase